src/apps_plugin.c

```diff
@@ -824,13 +867,15 @@ int read_proc_pid_stat(struct pid_stat *p) {
 
 	// ----------------------------------------
 
-	int set_quotes = (!ff)?1:0;
+	// FIX: set_quotes will always be true!
+	//int set_quotes = (!ff)?1:0;
 
 	ff = procfile_reopen(ff, filename, NULL, PROCFILE_FLAG_NO_ERROR_ON_FILE_IO);
 	if(!ff) return 1;
 
 	// if(set_quotes) procfile_set_quotes(ff, "()");
```

Comment:
these calls say to procfile which are the quote characters when parsing the files. In the case of `/proc/pid/stat` the `()` are quotes for the process name, which may contain spaces.
We set the quotes just once, after allocating `ff`.